hub/constants.py

```diff
@@ -10,7 +11,33 @@
 
 DEFAULT_HTYPE = "generic"
 
-SUPPORTED_COMPRESSIONS = ["png", "jpg", "jpeg", None]
+SUPPORTED_COMPRESSIONS = [
```

Comment:
is this the intersection of those 2 lists you were talking about? if so, could you add a comment here

Reply:
There are hard coded values.. but we filter them below, in case plugins for any of them are not installed.